cypress/utils/dashboards/vis_type_table/buckets_agg.js

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Terms Aggregation
+ */
+
+Cypress.Commands.add('toggleOtherBucket', (request) => {
```

Comment:
Can we add prefixes to these custom commands? I can see other commands being named similarly. e.g for VisBuilder specific commands i prefixed them with `vb` so that there arent two commands with the same name.

Reply:
Good call. I didn't realize that. I will update the command names.